app/controllers/application_controller.rb

```diff
@@ -45,6 +46,14 @@ def check_team_authorization(team_id: params[:team_id])
     redirect_to root_path, status: :forbidden
   end
 
+  def check_member_authorization(user_id: params[:user_id])
+    @member = Member.new(user_id)
```

Comment:
```suggestion
    member = Member.new(user_id)
```
doesn't need to be an instance variable?
also, probably a guard clause (checking `params[:user_id]` is set) should be here, to stop creating the object for every single request

Reply:
@jakubmiarka very good point! I have amended :)